helpers.go

```diff
@@ -0,0 +1,23 @@
+package par
```

Comment:
My rules of thumb for where to place helpers:
- If they're only used by one file in a package, they belong in that file. Otherwise a separate file.
- If they're only used by three packages, each package gets a copy. Otherwise a separate package.

Could you move the helpers to `par.go`? Being an open source package, providing a single file also has the benefit of making the package simpler to pull into a project with some minor changes without needing to formally fork, as all you need to copy into your project is a single file.